apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/plugin/loader/AgentClassLoader.java

```diff
@@ -77,6 +77,7 @@ public static void initDefaultLoader() throws AgentPackageNotFoundException {
             synchronized (AgentClassLoader.class) {
                 if (DEFAULT_LOADER == null) {
                     DEFAULT_LOADER = new AgentClassLoader(PluginBootstrap.class.getClassLoader());
+                    Thread.currentThread().setContextClassLoader(DEFAULT_LOADER);
```

Comment:
This is only work for current thread, why this effects the Kafka kernel? How they share the same thread?